roles/keystone/tasks/main.yml

```diff
@@ -72,12 +72,19 @@
 
 - name: configure keystone
   template: src={{ item }} dest=/etc/keystone/
-            mode={{ 0644 if 'policy.json' in item else 0640 }}
+            mode=0640 
```

Comment:
The policy.json file was removed from the keystone role so i removed the check here.